docs/sql/mysql/create_schema.sql

```diff
@@ -239,7 +239,7 @@ CREATE TABLE tpp_user_consent (
   user_id               VARCHAR(256) NOT NULL,
   client_id             VARCHAR(256) NOT NULL,
   consent_id            VARCHAR(64) NOT NULL,
-  external_id           VARCHAR(256) NOT NULL,
+  external_id           VARCHAR(256),
```

Comment:
We use `external_transaction_id` in table `ns_operation`, if this is the same identifier, perhaps the naming should be also consistent. Otherwise ignore this comment.

Reply:
We will keep as is.